pycsw/core/metadata.py

```diff
@@ -1241,9 +1241,22 @@ def _parse_iso(context, repos, exml):
             _set(context, recobj, 'pycsw:ConditionApplyingToAccessAndUse',
             md.identification.uselimitation[0])
 
-    if hasattr(md.identification, 'format'):
-        _set(context, recobj, 'pycsw:Format', md.distribution.format)
-
+        if hasattr(md.identification, 'format'):
```

Comment:
The changes to `pycsw.core.metadata` are not relevant to the subject of this pull request. Therefore they should not be a part of it. Please remove them from the PR. If you wish to also propose these changes to pycsw please submit a different PR with the changes.